package.json

```diff
@@ -24,21 +25,32 @@
     "bootstrap": "3.3.7",
     "enzyme": "2.9.1",
     "eslint": "4.3.0",
+    "eslint-config-nava": "^2.0.0",
     "gh-pages": "1.0.0",
     "jquery": "3.2.1",
     "pretty-data": "0.40.0",
+    "prop-types": "^15.5.10",
     "qpp-style": "1.4.2",
     "react": "15.6.1",
     "react-copy-to-clipboard": "5.0.0",
     "react-dom": "15.6.1",
     "react-file-download": "0.3.2",
+    "react-router-dom": "^4.1.2",
     "react-scripts": "1.0.10",
     "react-tabs": "0.8.3",
     "react-test-renderer": "15.6.1",
-    "uswds": "1.0.0",
-    "react-router-dom": "^4.1.2"
+    "uswds": "1.0.0"
   },
   "eslintConfig": {
-    "extends": "react-app"
+    "extends": [
+      "nava",
+      "react-app"
+    ],
+    "parserOptions": {
+        "ecmaVersion": 6,
+        "ecmaFeatures": {
+            "jsx": true
```

Comment:
Maybe I'm misunderstanding, but won't this option mean that we will be linting `.jsx` files so we don't need to change their extensions to `.js`? Or does this option just mean it will validate the JSX syntax but won't necessarily find `.jsx` files?